README.md

```diff
@@ -1,32 +1,36 @@
-# Joomla! X Clean Base Branch
+# Joomla! X Doctrine Branch
 
-This branch contains the current development of Joomla! X (currently, X == 4),
-based on a clean-base approach.
+# This branch is pre alpha, it is far from finished!!
 
-## Why Clean Base?
+## Todo's
+Some more work needs to be done:
 
-We honestly tried an incremental approach, i.e., to transform J!3 into J!4 using small steps.
-The tight coupling and non-sufficient test coverage, however, makes it extremely hard to apply a state-of-the-art architecture.
+- Connection sharing
+- Better test coverage
+- Real world example
+- Doctrine finder's need to work per entity
 
-It was suggested to start from a fresh ground, the clean base, to create the basic architecture,
-and to port existing functionality in a second step.
-The J!4 Architecture Team decided to try this approach.
+## Installation
+To make it running execute the following command on your command line within the root directory:
 
-## The "Chris Davenport Happy Milestone" (CDHM)
+`composer install`
 
-The team defined a milestone, named after the team member, Chris Davenport, who brought up the requirements.
-You can see the original document here: [docs/j4cdhms.md](docs/j4cdhms.md).
+## Demo
 
-The final decision is made, when the milestone is reached, or it shows out, that it can not be reached with the clean base approach.
+#### Setup
+We prepared a little script which creates a sqlite database file with an article table and one entry with the id 1. On the command line run:
 
-## Installation
+`php installation/demo.php`
+
+This will create the database file demo.sqlite in the installation folder. Make sure the database settings in the file config/database.ini do pint to that file.
+
+#### Browser
+Open your browser with the following url:
 
-As there is no installer yet, a manual installation is required.
-Check out this branch (`clean-base`), and run
+*http://{{address}}/{{path}}/?id=1&entity=article*
 
-```bash
-$ composer install
-```
+## PHPUnit
+You can run the tests with codeception like `./libraries/vendor/bin/robo test:unit` or good old `phpunit`.
 
 ## Contribute
 
```

Comment:
Don't change the README - add your text to the description of the PR instead. Otherwise, a merge would destroy the branch's README.